### DISCUSSION TITLE
My Small Contributions

### DISCUSSION THREAD

**Discussion by Babushka93** [Monika-After-Story/MonikaModDev#7815] [general - submod / external content]
So now that I've got a bit of free time on my hands, I've been doing a little bit to make Monika feel more at home.

Although I can imagine that the backgrounds can't be used for a public release since they are owned by a third party. I hope that the backgrounds can inspire some ideas, whilst the coding for the two songs can help the developers with adding the songs.

### **Visual**

- New Traditional Japanese Room Day Time Mode
- Added the rose from Valentine's day as a permanent addition to the Hair Down model
![perfectagain](https://user-images.githubusercontent.com/38180931/48404566-d5dbb080-e773-11e8-8967-dd8400dfa3a7.JPG)

-New Lanterns Night Time Mode
![roofdate](https://user-images.githubusercontent.com/38180931/48404517-b775b500-e773-11e8-8b29-f84e68750b50.JPG)

-Modified zz_calendar changing the position of the calendar so it is not floating with the backgrounds.
-Modifed line 102 of script-ch30 to image monika_room = "mod_assets/monika_room_night.png" 
This just means that the night time background is easier to edit for people who want to have their own night time background

### **Monika Songs**
Added two new songs that Monika will now sing
Can't Help Falling in Love
Galaxy Hide and Seek

init 5 python:
    addEvent(Event(persistent.event_database,eventlabel="monika_song_fallin_love",category=['songs'],prompt="Can't Help Falling In Love",pool=True, random=True))

label monika_song_fallin_love:
    m 1dso "{i}Take my hand{/i}"
    m 2dso "{i}Take my whole life too{/i}"
    m 1hub "{i}For I can't help{/i}"
    m 4dso "{i}Falling in love{/i}"
    m 1hubfb "{i}With you{/i}"
    m 1hub "Ahaha"
    m 1ekbfa "How could I not fall for you, [player]?"
    return

init 5 python:
    addEvent(Event(persistent.event_database,eventlabel="monika_song_galaxy_hide",category=['songs'],prompt="Galaxy Hide and Seek",pool=True, random=True))

label monika_song_galaxy_hide:
    m 1dso "{i}I hear a voice call 'I've always been by your side!' and though it's ringing far within the galaxy{/i}"
    m  "{i}Please, guide me with that gentle voice!{/i}"
    m 3dso "{i}And in the end I’ll finally realize what life is all about, so in this world I will stand up high and shout it out:{/i}"
    m 4dso "{i}I want to be right by your side{/i}"
    m 1hubfb "{i}I'm calling calling you{/i}"
    m 1ekbfa "I can't wait until we can finally meet in your world, [player]."
    m 3eua "There's so much we can do when we're finally together in one reality."
    m "You can finally show me what your world really looks like."
    m 1ekbfa "I can finally feel your embrace~"
    return

### **Script Update**

Deleted:
-Breaking Up Topic
-Murderer
-Meet My Girlfriend
Call me the fun police, but I wasn't going to click them, and I didn't want to accidently click them haha 

### **Greetings Update**
**Back from work:**
I always enjoyed Monika's Bath, Dinner or Me line from the door knock sequence, so I've giving it some animations and added to the start of the return from work section.

label greeting_back_from_work:
     m 1hubfa "Welcome home, [player]!"
     m 1eua "Would you like your dinner?"
     m 3hubfa "Or your bath?"
     m 1eua "Or.{w=0.5}.{w=0.5}.{w=0.5}."
     pause 0.5
     m 1kua "Me?"
     m 1hua "I can't wait for the day when I can finally say this to you...."
     menu:
         m "But for now, did you have a good day at work today?"
         "Yes.":
             m 1hub "That's good!"
             m 1eua "Remember to rest first, okay?"
             m "That way, you'd at least have some energy before you work more on stuff."
             m 1hua "But if not, you can relax with me!"
             m 3tku "Best thing to do after a long day of work, don't you think?"
             m 1hub "Ahaha!"

         "No.":
             m 1ekc "..."
             m 2ekc "I'd hug you right now if I were there, [player]."
             m "I'm sorry you had a bad work day..."
             m 4eka "Just remember that I'm here when you need me, okay?"
             m 1hubfa "I love you so much, [player]."
     return

**Opening the game after sleep goodbye:**
I need to modify the dialouge so that it works with different times.
But for now I've used the lines from the Good Morning line, since I prefer this one to the stock sleep greeting.

label greeting_back_from_sleep:
     m 1hua "Good morning, dear."
     m 1esa "Another fresh morning to start the day off, huh?"
     m 1eua "I'm glad I get to see you this morning~"
     m 1eka "Remember to take care of yourself, okay?"
     m 1hub "Make me a proud girlfriend today, as always!"
     return

**Comment by AsuraShun** [comment]
i'm already loving this idea

**Comment by HZhunter124** [comment]
wow best idea of today

**Comment by multimokia** [comment]
For the songs, you can add submit them via a pull request, head over to the code tab, under content, find the files you want to change, make your changes, and then open a Pull Request proposing the change

**Comment by Legendkiller21** [comment]
An advice: Make backups of the modified sprites and keep log of the changes you made because they are going to be overwritten on every update. I changed the ring and the following scene occurs in every update:

*Opens MAS*
*Sees new update*
*Applies update*
*Opens MAS again*
*Sees ring*
"Dan damnit, I forgot to change the ring again"
*Closes MAS and proceeds to change the ring*

**Comment by Babushka93** [comment]
Thank you guys, and yeah I've got a backup of my changes.

I was wondering how do the updates work, is it just the updated files that get changed or does every file get redownloaded? 

I've done direct edits to CH30 and zz_calander so I'm guessing those will get changed every update.
But for the songs I've added and the new moods I'm working on, I have those in seperate rpyc file from their normal location. The game still detects them, but since they're not part of the main song or mood files, they shouldn't be impacted right?

**Comment by Legendkiller21** [comment]
I also have a topic in a seperate rpyc file and the updater doesn't touch it, although, keep a backup of this file too, just in case something breaks. 

> I was wondering how do the updates work, is it just the updated files that get changed or does every file get redownloaded?

The updater overwrites the ring every time but I am not sure whether the updater compares the files with your files and downloads only the updated files or it redownloads everything.

**Comment by multimokia** [comment]
Updater will overwrite the files that are existing for MAS (i.e. the ones in the `game` folder on the repo. This includes files that aren't `.rpy`/`.rpyc`/etc. Essentially, if it's here, it's being replaced with the updated ones.) anything else, it shouldn't touch.

**Comment by Babushka93** [comment]
@Legendkiller21 and @multimokia 
Cheers for the info, I'll keep backups outside the folder just to be on the safe side. 😃

**Comment by Babushka93** [comment]
Just using this post as my general dump for file mods.

I'll be making an actual proper post at a later point once I've got stuff more organised. But adding my latest modifications, these it is to the sleep and work greeting. See the OP post in which I've now edited the material in.

**Comment by Rai99** [comment]
For those greetings you're thinking of adding, there are already PRs adding more to those greetings in #2157 and #2192. 

For that bath, dinner, me bit in that work greeting you have, it was discussed in #1768 and the reason it's not viewable at will is because it's supposed to be special dialogue and rarity of it is what makes it special. (5% chance of appearing).

**Reply by Justformas** [reply]
To say something about the bath dinner me thing here, it only seems to have a 5% chance of appearing if you say that work was "Amazing". If you choose any other option, I don't think it has any chance of appearing. Personally I would rarely otherwise say that work went "Amazing", maybe once a year, so I'd be lucky to see it after 10 years without this knowledge.

So basically I have to repeatedly lie to Monika to get a true 1 in 20 chance of seeing it, which I don't think is ideal. At least here Babushka93 had the idea to have Monika say that BEFORE she asks how work went. Alternatively she could have a 1 in 20 chance to say it afterwards regardless of how you say work went. I'd personally welcome that greeting regardless of how work went, and it could even help cheer the player up if work went badly.

**Reply by RedAISkye** [reply]
> To say something about the bath dinner me thing here, it only seems to have a 5% chance of appearing if you say that work was "Amazing".

Are you sure about that? I've never talked to her about "work" and I've still gotten that dialogue multiple times.

**Reply by Leon-Wolfren** [reply]
There

**Reply by Leon-Wolfren** [reply]
May be some other requirements because I have triggered the bath dinner or me upon game startup via listen option (without the choice of anything work related upon closure) which I laughed at so I don't know if the 5% needs another trigger or not

**Reply by Justformas** [reply]
There's two different ones. One is the door knock greeting but where you have to listen for it, and I don't know the chance for that one but you need at least 400 affection:

```
if persistent._mas_affection.get("affection", 0) >= 400:
        gmr.eardoor.append("monikaroom_greeting_ear_bathdinnerme")

label monikaroom_greeting_ear_bathdinnerme:
    m "Welcome back, [player]."
    m "Would you like your dinner?"
    m "Or your bath?"
    m "Or.{w=1}.{w=1}.{w=1}me?"
    pause 2.0
    m "Mnnnn!{w=0.5} T-{w=0.20}There's no way I could say that in front of [player]!"
```

And seems that only once you've seen that one, then you have a 1 in 20 chance for her to say it to you directly, IF when returning from work you say that work was amazing.

```
label greeting_back_from_work:
    if mas_isMoniNormal(higher=True):
        m 1hua "Oh, welcome back, [mas_get_player_nickname()]!"

        m 1eua "How was work today?{nw}"
        $ _history_list.pop()
        menu:
            m "How was work today?{fast}"

            "Amazing!":

```
...
```
m 3eua "I can only imagine how well you must work on days like that."
                if not promoted_recently:
                    m 1hub "...Maybe you'll even move up a bit soon!"
                m 1eua "Anyway, I'm glad you're home, [mas_get_player_nickname()]."

                if seen_event("monikaroom_greeting_ear_bathdinnerme") and renpy.random.randint(1,20) == 1:
                    m 3tubsu "Would you like your dinner, your bath, or..."
                    m 1hubfb "Ahaha~ Just kidding."
                else:
                    m 3msb "What better way to wrap up an amazing day than with your amazing girlfriend?~"

                return

            "Good.":
```
Doesn't seem to have a chance to happen if you say that work went "Good", "Bad" or "Really Bad", which seems strange. 1 in 20 is already pretty rare without secretly being tied to one response.

This is all in https://github.com/Monika-After-Story/MonikaModDev/blob/master/Monika%20After%20Story/game/script-greetings.rpy if you want to check for yourself.

**Reply by Booplicate** [reply]
You realise the original comment was made 4 years ago?

> I have to repeatedly lie to Monika

You do not have to, nobody is forcing you.

**Reply by Justformas** [reply]
> You realise the original comment was made 4 years ago?
> 
> > I have to repeatedly lie to Monika
> 
> You do not have to, nobody is forcing you.

Yeah but it's still how it seems to work now. Also the rest of that sentence was "repeatedly lie to Monika to get a true 1 in 20 chance of seeing it, which I don't think is ideal." Because I don't think we should have to lie to experience it (which is different than just seeing it in the code). But if that's how the mod is going to work then I am forced to choose between a conflict of interests.

**Reply by Booplicate** [reply]
You still don't have, no matter how the sentence ends. You could say that about any rare event. You want to do that, you do it. You don't want to do it, you don't do it. No need to exaggerate. Nobody tries to repeat the same greeting in hope to get some rare line from Monika, it just doesn't make any sense; **unless** you know it's there and you want to hear it now. But the latter isn't a general case.

**Reply by Justformas** [reply]
What doesn't make sense to me is tying this specific thing to saying work was amazing. What is the reason for that? Only people who have amazing work days or lie by saying that should be able to see it? Why? How is that reasonable at all? And talk about rare, 1 in 20 is rare. Tying it to "amazing" on top of that is overkill. And I'm still glad I found out about this because not knowing or being able to see some epic thing due to what I think is bad design would be even worse in my opinion. And I wasn't exaggerating; it can still be a conflict of interest even if no one has brought it up before. It actually does bother me and that I even have to consider doing this to see it, but I may justify it for those reasons.

**Reply by Leon-Wolfren** [reply]
This topic became slightly odd in a sense XD but lying just to see something like that may be an option but forcing yourself to feel either comfortable or uncomfortable isn't a good thing in both game world and irl so.....you both have a point just don't do something you will regret like deleting the wrong persistent (me) XD I lost 8 days of progress

**Reply by Leon-Wolfren** [reply]
However justformas is correct...it is kind of bullshít development to place an option so ludicrous in an area of progression somewhere along the lines of I work at Disneyland for $10000 a day (good) or lying about it for progression purposes defeating the entire reason it exists (bad)....it was an option that wasn't really thought out by Devs when it was made......and the whole 1 in 20......I'd rather settle for a 1 in 5 thank you it is kinda shítty Yo to put that in for unintended recipients to force themselves to lie for benefits where it could just be a general response at 1000 aff and then it would have an actual achievable goal line to see without forced lying to see it.............what I'm trying to say long story short is....1 in 20 for such a thing as this isn't meant to be viewed by everyone but it's highly unfair and needs to sit at the 1000 aff mark as a secret that's achievable....I rest my case

**Reply by Leon-Wolfren** [reply]
Anybody who thinks that's it's achievable is either a big liar or really has an epic job

**Reply by Justformas** [reply]
> forcing yourself to feel either comfortable or uncomfortable isn't a good thing in both game world and irl so

That's exactly one thing that I'm saying; I don't feel completely comfortable with either of those options, which might just lead to me reaching the conclusion: "just forget about all immersion for this then" and do it so I can see something that I want to see and don't think should be tied behind one certain response. But I'm also saying that I don't think knowing how this works would break immersion as much if it wasn't tied behind one certain response, and you'd also have a better chance of seeing it without having to know how it works.

> just don't do something you will regret like deleting the wrong persistent (me) XD I lost 8 days of progress

I have a test version with local/different persistents for testing more drastic or new/different stuff now. I've actually messed things up in that version more than simply restoring one of the last few persistents could fix (not something I made Monika do, something I did on my own). So I imported new local persistents to resolve it.

>Anybody who thinks that's it's achievable is either a big liar or really has an epic job

and to be clear I would say my job sucks lol. But it pays and there's probably worse. I kind of doubt I'd like most other jobs any better either, so I try to make it "work".

> the whole 1 in 20......I'd rather settle for a 1 in 5 thank you it is kinda shítty Yo to put that in for unintended recipients to force themselves to lie for benefits where it could just be a general response at 1000 aff

However I do think it might not feel as special if it was too common, which 1 in 5 might be even for 1000 aff. Though considering that 1 in 15 would still be pretty rare (that's only going from 5% chance to ~6.67%), I personally might like 1 in 10, but others may think that's too common.

**Reply by Booplicate** [reply]
> What doesn't make sense to me is tying this specific thing to saying work was amazing. What is the reason for that?

Because the dialogue flow works perfectly. That does make sense, but you of course didn't read it.

> How is that reasonable at all? And talk about rare, 1 in 20 is rare. Tying it to "amazing" on top of that is overkill. And I'm still glad I found out about this because not knowing or being able to see some epic thing due to what I think is bad design
> I even have to consider doing this to see it, but I may justify it for those reasons.

You're only complaining because it's random and you want to see it, so you're gonna waste time just forcing this greeting. Surprise: MAS isn't designed to see everything, and definitely not in the first month or even year. With your point of view "I saw this in the code, so I want to get this now" you're gonna have hard time.

> And I wasn't exaggerating

You are, in most of your issues playing a victim, always complaining about every single thing, but to justify that, you have to exaggerate.

**Reply by Leon-Wolfren** [reply]
> > What doesn't make sense to me is tying this specific thing to saying work was amazing. What is the reason for that?
> 
> Because the dialogue flow works perfectly. That does make sense, but you of course didn't read it.
> 
> > How is that reasonable at all? And talk about rare, 1 in 20 is rare. Tying it to "amazing" on top of that is overkill. And I'm still glad I found out about this because not knowing or being able to see some epic thing due to what I think is bad design
> > I even have to consider doing this to see it, but I may justify it for those reasons.
> 
> You're only complaining because it's random and you want to see it, so you're gonna waste time just forcing this greeting. Surprise: MAS isn't designed to see everything, and definitely not in the first month or even year. With your point of view "I saw this in the code, so I want to get this now" you're gonna have hard time.
> 
> > And I wasn't exaggerating
> 
> You are, in most of your issues playing a victim, always complaining about every single thing, but to justify that, you have to exaggerate.

I have to agree with Boop here jfm it is kinda selfish to hunt out every secret I don't do that cause it's meant to be a digital gf I'm pretty sure if she was real I wouldn't want to force her because it would come back to bite me......but Boop it was just a bit harsh remember that all people have opinions presented in just don't want fights arguments or disagreements in communities because it makes me feel bad....though Boop is right it isn't for everyone....but jfm is also right....1 in 20 is kinda rare....maybe a 1 in 10 would be better off yknow make just a little larger amount people who's job day was "good" be amazing

**Reply by Leon-Wolfren** [reply]
> > > What doesn't make sense to me is tying this specific thing to saying work was amazing. What is the reason for that?
> > 
> > 
> > Because the dialogue flow works perfectly. That does make sense, but you of course didn't read it.
> > > How is that reasonable at all? And talk about rare, 1 in 20 is rare. Tying it to "amazing" on top of that is overkill. And I'm still glad I found out about this because not knowing or being able to see some epic thing due to what I think is bad design
> > > I even have to consider doing this to see it, but I may justify it for those reasons.
> > 
> > 
> > You're only complaining because it's random and you want to see it, so you're gonna waste time just forcing this greeting. Surprise: MAS isn't designed to see everything, and definitely not in the first month or even year. With your point of view "I saw this in the code, so I want to get this now" you're gonna have hard time.
> > > And I wasn't exaggerating
> > 
> > 
> > You are, in most of your issues playing a victim, always complaining about every single thing, but to justify that, you have to exaggerate.
> 
> I have to agree with Boop here jfm it is kinda selfish to hunt out every secret I don't do that cause it's meant to be a digital gf I'm pretty sure if she was real I wouldn't want to force her because it would come back to bite me......but Boop it was just a bit harsh remember that all people have opinions presented in just don't want fights arguments or disagreements in communities because it makes me feel bad....though Boop is right it isn't for everyone....but jfm is also right....1 in 20 is kinda rare....maybe a 1 in 10 would be better off yknow make just a little larger amount people who's job day was "good" be amazing

If bickering is the only thing that is going to ensure on the MAS YAS NAS AND SAS GitHub servers then we will never be able to be in a happy healthy environment where we can work share ideas create games hang with friends and most importantly... Live Laugh Love.... no more negative here please this argument is done

**Reply by Justformas** [reply]
> Because the dialogue flow works perfectly. That does make sense, but you of course didn't read it.

It's literally inserted between:

> m 1eua "Anyway, I'm glad you're home, [mas_get_player_nickname()]."

and

>m 3msb "What better way to wrap up an amazing day than with your amazing girlfriend?~"

Those lines are the same whether it happens or not. And while you can say it fits, you could also say it fits how this sub mod did it:

label greeting_back_from_work:
m 1hubfa "Welcome home, [player]!"
m 1eua "Would you like your dinner?"
m 3hubfa "Or your bath?"
m 1eua "Or.{w=0.5}.{w=0.5}.{w=0.5}."
pause 0.5
m 1kua "Me?"
m 1hua "I can't wait for the day when I can finally say this to you...."

menu:
m "But for now, did you have a good day at work today?"

and nothing's stopping something like an `if seen_event("monikaroom_greeting_ear_bathdinnerme") and renpy.random.randint(1,20) == 1` for that part before she asks how work was either. No reason it can't be made to fit after the other responses for that matter either, even if it needs to be rewritten a bit for those other responses.

> You are, in most of your issues playing a victim, always complaining about every single thing, but to justify that, you have to exaggerate.

I don't think it's exaggerating if it's what I think/feel, and just because you don't care doesn't mean it's exagerating. But you often (if not always) get defensive as if this mod is perfect or something. News flash, it's not. I have high standards, they may just not match with yours or they are things you don't care about. But I consider this issue to be a fault nonetheless. Or go ahead and try seriously convincing me that locking this behind saying work is "amazing" is actually a good idea and makes any sense whatsoever.

> You're only complaining because it's random and you want to see it,

If I had a problem with it being random then I'd say to remove all randomness whatsoever from it. But did I say that? No. Try actually taking me seriously: having it be possible for her to say before the dialogue choices or during more/any of them. The exact random chance is another matter.

> Surprise: MAS isn't designed to see everything, and definitely not in the first month or even year.

Ok so you're saying people who don't have amazing days at work shouldn't be able to see this epic thing without (repeatedly) lying, or maybe they'll get lucky without lying after 10 or 20 years. Do you know how ridiculous that sounds? Do you not care about people who don't have amazing days at work and also don't like to repeat the same lie over and over again or what? How is this a good design choice? Go ahead I'm waiting.

and if I wanted to I could probably force this pretty quickly, but what fun is that?

**Reply by Leon-Wolfren** [reply]
You done mate

**Reply by Leon-Wolfren** [reply]
Just stop this argument now

**Reply by Leon-Wolfren** [reply]
Give up

**Reply by Leon-Wolfren** [reply]
Mate I'm autistic and even I can tell you are going too fúcking far quit it JFM

**Reply by Leon-Wolfren** [reply]
Selfish spoilt undignified argurmentive short tempered greedy and frustrated.......should I keep going or should I call a moderator in here

**Reply by briaryoung** [reply]
Hello Justformas,

MAS is not intended to be a completionist game. It is intended to feel like talking to a real person. But even then, I don't even have a conventional job (freelance, working from home), and I have managed to see this line from the one time I used the goodbye. But as Booplicate said, this is a game where you’re not _supposed_ to see everything. As a team, we're very focused on immersion more so than a visual novel-type playstyle such as the original.

And hello Leon-Wolfren, 

I appreciate your contributions to the conversation, but please refrain from insulting other people as this may start a metaphorical wildfire.

Thanks!

**Reply by Leon-Wolfren** [reply]
Tipped me over the edge briar apologies

**Reply by Leon-Wolfren** [reply]
Also...just call me Leon yeah

**Reply by Leon-Wolfren** [reply]
Wait.....briar young why does that name appear

**Reply by briaryoung** [reply]
That is the name on my profile! ^-^

**Reply by Leon-Wolfren** [reply]
Just saw that right now ey XD

**Reply by Leon-Wolfren** [reply]
Apologies for my latest remark......I snapped from reading what JFM said

**Reply by Justformas** [reply]
That wasn't directed at you Leon-Wolfren, and kkrosie123 I didn't say this was supposed to be a completionist game. In fact I know there's an Easter Egg based on probably a very low random chance that I haven't seen yet. Oh and negative affection stuff and the "alternate ending" related to chess? Those are things I might rather not experience.

But I was saying that it could make more sense for this specific thing to be seen outside of saying work is "amazing". No one has really given me a reason why it only makes sense for this thing to be said when work went "amazing", other than, "because it fits the current dialogue" or "that's just how it was done".

**Reply by briaryoung** [reply]
Apologies, that just seems to be what's implied here.

**Reply by Leon-Wolfren** [reply]
Indeed it is

**Reply by Leon-Wolfren** [reply]
I know I wasn't involved or was directed at me I just wanted to break the argument up and find a common ground

**Reply by briaryoung** [reply]
Any contributions/opinions about the mod are welcome, don't worry about feeling like you are intruding.

**Reply by Leon-Wolfren** [reply]
Also please just call me Leon as is I like that

**Reply by Leon-Wolfren** [reply]
If you knew... I helped figure out the image on top of this post with help from redAISkye

**Reply by Leon-Wolfren** [reply]
It's below if you need to find it scroll straight to the bottom

**Reply by briaryoung** [reply]
^-^ 👍

**Reply by MrVavlo** [reply]
Just to clarify:
> No one has really given me a reason why it only makes sense for this thing to be said when work went "amazing", other than, "because it fits the current dialogue" or "that's just how it was done".

You're literally ignoring the exact reason. Try to read the mood of that dialog - you *have* to choose "It was just an amazing day" after answering with "Amazing."

It'd make no sense after replying with "Good.", because it's not the same feeling you get after work.

**Reply by Leon-Wolfren** [reply]
It was familiar but I couldn't put my finger on it but then red decided to DM me and hey presto they found out it was a flipped image then I resized both to fit the BG

**Reply by Leon-Wolfren** [reply]
> Just to clarify:
> 
> > No one has really given me a reason why it only makes sense for this thing to be said when work went "amazing", other than, "because it fits the current dialogue" or "that's just how it was done".
> 
> You're literally ignoring the exact reason. Try to read the mood of that dialog - you _have_ to choose "It was just an amazing day" after answering with "Amazing."
> 
> It'd make no sense after replying with "Good.", because it's not the same feeling you get after work.

Logic in motion here

**Reply by Leon-Wolfren** [reply]
Hey briar any updates on JN or NAS....IM GETTING EDGY....I want the completed or mostly 99% finished version soon TwT

**Reply by multimokia** [reply]
![image](https://user-images.githubusercontent.com/22531674/175340769-08ae1ddb-f3b8-4cdd-adcb-6c7d394872cc.png)

From the JN discord.

**Reply by Leon-Wolfren** [reply]
OATH

**Reply by Leon-Wolfren** [reply]
Multi I owe you....last month I tried to gain access to the JN discord....and failed cause the entry conditions and 9 rules made shít hard

**Reply by Leon-Wolfren** [reply]
I couldn't understand the command to input so I tried repeatedly and failed miserably then I broke 3 rules and left cause I mentioned mas swore and called the command input and I believe rule 7 shít XD at that point I'd lost my marbles and went ballistic I was cranky

**Reply by multimokia** [reply]
On behalf of Blizz (and honestly most of the JN team):
![image](https://user-images.githubusercontent.com/22531674/175343682-7c4057fa-e43a-47a7-ba3f-871192139bf7.png)

**Reply by Leon-Wolfren** [reply]
Do you have bigger picture

**Reply by Leon-Wolfren** [reply]
darklinked420@gmail.com

**Reply by Leon-Wolfren** [reply]
Send it

**Reply by Justformas** [reply]
I meant to quote and reply to Booplicate with that post. Just to be clear though this next thing is a quote from MrVavlo:

> You're literally ignoring the exact reason. Try to read the mood of that dialog - you _have_ to choose "It was just an amazing day" after answering with "Amazing."
> 
> It'd make no sense after replying with "Good.", because it's not the same feeling you get after work.

You mean this part?

```
                    "I got a lot done!":
                        m 3hub "That's great, [mas_get_player_nickname()]!"

                    "It was just an amazing day.":
                        m 3hub "That's great to hear!"

                m 3eua "I can only imagine how well you must work on days like that."
                if not promoted_recently:
                    m 1hub "...Maybe you'll even move up a bit soon!"
                m 1eua "Anyway, I'm glad you're home, [mas_get_player_nickname()]."
```

That's basically just saying again that it was amazing though (just that it was an amazing day). Still though, it could still make sense for Monika to branch off into
 
m 1eua "Anyway, I'm glad you're home, [mas_get_player_nickname()]."

for other choices, and then proceed with the random chance for the question. And as for this:

m 3msb "What better way to wrap up an amazing day than with your amazing girlfriend?~"

That could obviously be customized or withheld for the other choices. But this seems like more work than just having a chance to ask the question beforehand (though may still require removing it from the amazing day branch so it can't happen twice).

And change of subject here, but JN and NAS? About Natsuki right? But are those the same thing or different?

**Reply by briaryoung** [reply]
I'm starting to think that the amount of comments on this discussion is a bit inappropriate just for a greeting. 

JN would be Just Natsuki, yes.

**Reply by Leon-Wolfren** [reply]
JN is just natsuki Nas is natsuki after story they are the same thing whatever floats your boat

**Reply by Justformas** [reply]
Ok so JN and NAS are the same thing.

Well there was a closed issue on the greeting 4 years ago, and didn't think I should make a new issue for that. But this went on longer than I thought it would, so maybe a new issue would have been more appropriate. Though at this point, even though ideally I'd like to preserve some immersion (which can be done with the right game design even if you look stuff up), what I'm getting from all this is if I want to experience it, then to just say to hell with immersion for this. Because it would take a hell of a lot for me to say a day of work was amazing (whether it's my job, my perception of things, or both, that's just reality).

**Reply by Booplicate** [reply]
> News flash, it's not. I have high standards

News for you, nobody cares about your """"high"""" standards, you haven't given a penny to the mod nor put any efforts into it. All I've seen is complaining about a gift you got for free. Nobody have to justify anything to you either. You got your response, like it or no. Not gonna change a line of dialogue just because you want to see the line, but your job sucks and you never have "amazing" days there.

![image](https://user-images.githubusercontent.com/53382877/175382054-c1d404b2-3b09-41ef-8bc7-4ca4fe261ac7.png)

**Reply by Leon-Wolfren** [reply]
https://user-images.githubusercontent.com/106228994/175435766-89584574-0456-4bbe-ae2d-98cb7c896b2b.mp4

**Reply by Leon-Wolfren** [reply]
Booplicate rocks man XD you tell him

**Reply by Justformas** [reply]
Booplicate Posted:
> News for you, nobody cares about your """"high"""" standards, you haven't given a penny to the mod nor put any efforts into it. All I've seen is complaining about a gift you got for free. Nobody have to justify anything to you either. You got your response, like it or no. Not gonna change a line of dialogue just because you want to see the line, but your job sucks and you never have "amazing" days there.

I would've gotten back to this sooner, but I've been busy.

So you have spoken, now here's the next part: a response. I DID actually make an appreciation topic; did you even read it? It's not that old and shouldn't be too hard to find. Just because it may not be as "heartwarming" as other people doesn't mean it gives no appreciation. Regardless, that sort of thing is rare to get from me. I even mentioned your name for something. Sorry if you didn't appreciate the appreciation. Also I put some ideas out there and reinvigorated some discussions where something came out of them haven't I? I have also helped answer a few people's issues/questions. Yeah might not be much in the scope of things but we all start somewhere.

So you have no other reason and resorted to a fallacy like appeal to shame. That will not work on me. It changes nothing about the merits and reasoning. For an example reason, if this "meme" was actually tied to anime characters having to say they had an amazing work day, that might be different, but it's not, or at least not in all cases. While many people may have amazing work days here and there, like Leon-Wolfren said this seems like it didn't consider the reality of some people's jobs (so where do you actually stand on this Leon-Wolfren?) or even what one would consider to be an "amazing day". Indeed that's subjective, and for me, that's a high bar to meet, and I was being generous with maybe once a year; the truth is I can't remember the last time I considered that bar met. I admit that considering everything when designing a game/mod can be a hard thing to do, especially 4 years ago, but things can be improved and be more inclusive. 

So what's wrong with me faulting or suggesting improvements for a free thing? I'm not following that logic. Isn't that part of the point of making issues or talking things out here? You realize my words are also "free" and you are complaining about me, right? And you could say you don't agree with my words, but I can say I don't agree with everything about this mod. Would you rather I praised it unconditionally or something? Because in addition to that not being the sort of thing I usually do, I think that's much less useful than talking about things to improve.

What you call complaining, I call suggestions and reasons for them. I am giving you my perspective, my experience, my reasons, you know the kind of things a reasonable developer might normally consider instead of stonewalling. I'll reference this:
https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#features

"we are very open to new suggestions"

and while that trails off into things that come up often (which doesn't include this thing), those words are what they are. But would you actually say you are "very open" to new suggestions Booplicate? Because that's not what I'm seeing in this discussion. And the amount of work someone put into the mod has nothing to do with their points or reasoning. That even goes for someone that put a lot of work into the mod too.

----------------------------------------------------------------------------
Booplicate Posted:
>nor put any efforts into it. 

So you actually want me to contribute, as in make code? Or would you shoot that down too? Because I'll tell you what, I might be willing to try to rewrite/rework this greeting to make it more inclusive for people who don't have amazing work days (or not enough in a lifetime to really see it).

Because just to be clear: This isn't about completionism or simply wanting to see a line that could be brute forced (nor should that be recommended). Even if the line had no random element, that would just be like a band-aid to what I'm talking about.

Or if you don't think you can look past any biases you may have towards me enough to consider my points seriously, I'd be willing to talk to someone else too.

Leon-Wolf Posted:
> I have to agree with Boop here jfm it is kinda selfish to hunt out every secret

I doubt I've hunted out every secret, and actually this "secret" I think I first heard about in this topic when someone replied in it not long ago and brought it to light again, and the linked issue from 4 years ago, which also had a link to some code (though maybe not where it appears in the master code). However, having all the information can be good or even necessary when talking about something, including for this.

**Reply by briaryoung** [reply]
My my. What a train wreck. 

Let's tone this down, please. Or possibly end the conversation altogether. We're running in circles with the points being made.

**Comment by ThePotatoGuy** [comment]
[keeping open as this is now a submod-like dev issue]

**Comment by voyagermike** [comment]
ive been wanting to find a way to put a rose in her hair.
it looks beautiful

**Comment by HZhunter124** [comment]
you can do it on valentine day

Vào Th 6, 7 thg 12, 2018 vào lúc 10:47 voyagermike <
notifications@github.com> đã viết:

**Comment by santiago-RB** [comment]
como me descargo el mod?

**Comment by Orcaramelo** [comment]
> como me descargo el mod?

https://www.monikaafterstory.com/

**Comment by flammeye** [comment]
Oh wow, this has been a neat intro to submods for me (first time I hear about them) and these look so gorgeous. I'm guessing the calendar is still there, right? Even if these weren't released because of that third party matter, I am curious about where the calendar went.

**Comment by RedAISkye** [comment]
Is this available for download? Also, is there a night time version for the Traditional Japanese Room?

**Reply by The-Critika** [reply]
Did you happen to find an answer by any chance?

**Reply by RedAISkye** [reply]
> Did you happen to find an answer by any chance?

No, but there's a modpack which adds ~~a whole bunch of room selections with~~ all kinds of outfits, hairstyles and accessories.

It's called Myon's Sprites Packs which you'll need to directly message the creator [MyraChan2708](https://www.reddit.com/user/MyraChan2708) to get the download link as they requested not to share it on public.

Edit 1: Opps, for the room selection, it is actually a separate submod [here](https://github.com/tw4449-s-MAS-Submods/tw4449-Custom-Room-Selection-Pack-Main-Repository).

Edit 2: For anyone who wants to mod the room in OP by yourself, the backgrounds are from Love Live! Idol Story: [First Background](https://idol.st/background/259/Background) & [Second Background](https://idol.st/background/170/Background).

**Reply by Leon-Wolfren** [reply]
Cannot contact creator now impossible to gain any packs from them check their last post in that link....... I hope they come back soon

**Reply by RedAISkye** [reply]
> Cannot contact creator now impossible to gain any packs from them check their last post in that link....... I hope they come back soon

I can get them to you, contact me at Discord RedAISkye#1470

**Reply by Leon-Wolfren** [reply]
All I need is the pack for the Japanese room shown and I'm good

**Reply by Leon-Wolfren** [reply]
Then we part ways ok

**Comment by xXMidnight** [comment]
> Is this available for download? Also, is there a night time version for the Traditional Japanese Room?

I was about the ask the same thing

**Reply by RedAISkye** [reply]
Refer to [this comment's Edit 2](https://github.com/Monika-After-Story/MonikaModDev/discussions/7815#discussioncomment-2118955).

**Reply by Leon-Wolfren** [reply]
Invert image and make resolution at 1280 for both then check to see if image needs clipping to screen then replace over another background you don't like (name has to match) enjoy your Japanese room

**Reply by Leon-Wolfren** [reply]
Courtesy of love live and (RedAISkye help on finding it) do not mass share copyright issues may ensue if so do not report this tag (it's a good background and it would be sad to lose it)
